backend-lib/src/main/rust/lib.rs

```diff
@@ -797,9 +727,7 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_ge
         let addr = utils::java_string_to_rust(&env, address);
         let taddr = TransparentAddress::decode(&network, &addr).unwrap();
 
-        // We select all transparent funds including unmined coins, as that is the same
-        // set of UTXOs we shield from.
-        let min_confirmations = 0;
+        let min_confirmations = NonZeroU32::new(1).unwrap();
```

Comment:
There's a bug here: in the [iOS SDK](https://github.com/zcash-hackworks/zcash-light-client-ffi/blob/055b239fdd4218cdc9d7cbf354689f50891dc726/rust/src/lib.rs#L1339-L1346), I use `1` here and then use the `target` height instead of the `anchor` height to ensure I'm actually getting the zero-conf balance. We should make that fix here for now (so this is consistent with the iOS SDK), and then in a subsequent refactor we can migrate this API to `get_wallet_summary`.

Reply:
I couldn't align the Android SDK FFI implementation with the iOS side on the provided link. `NonZeroU32::MIN` and `anyhow!` caused compile errors. So I just changed the lines 737 and 738 only. See related commit and update if necessary. Thank you.

Reply:
@str4d Could you please prepare the necessary FFI changes here?

Reply:
Fixed.